core/src/main/java/com/sequenceiq/cloudbreak/repository/OrganizationResourceRepository.java

```diff
@@ -6,19 +6,25 @@
 import java.util.Set;
 
 import org.springframework.data.repository.NoRepositoryBean;
+import org.springframework.data.repository.query.Param;
 
 import com.sequenceiq.cloudbreak.aspect.DisableHasPermission;
+import com.sequenceiq.cloudbreak.aspect.DisabledBaseRepository;
 import com.sequenceiq.cloudbreak.aspect.organization.CheckPermissionsByOrganization;
+import com.sequenceiq.cloudbreak.aspect.organization.CheckPermissionsByOrganizationId;
 import com.sequenceiq.cloudbreak.domain.organization.Organization;
 import com.sequenceiq.cloudbreak.domain.organization.OrganizationAwareResource;
 
 @NoRepositoryBean
 @DisableHasPermission
-public interface OrganizationResourceRepository<T extends OrganizationAwareResource, ID extends Serializable> extends BaseRepository<T, ID> {
+public interface OrganizationResourceRepository<T extends OrganizationAwareResource, ID extends Serializable> extends DisabledBaseRepository<T, ID> {
```

Comment:
If you want to extend DisabledBaseRepository, DisabledBaseRepository's methods must be annotated like BaseRepository's.
like:
@CheckPermissionsByTarget(action = WRITE, targetIndex = 0) and etc

Reply:
I've changed back to BaseRepository.